PackageExplorer/MainWindow.xaml.cs

```diff
@@ -235,6 +301,42 @@ private void OnPackageViewModelPropertyChanged(object sender, PropertyChangedEve
                     Content = RootLayout;
                 }
             }
+            else if (e.PropertyName == nameof(PackageViewModel.PackageSource))
+            {
+                if (File.Exists(viewModel.PackageSource))
+                {
+                    InitPackageFileWatcher(viewModel.PackageSource);
+                }
+            }
+            else if (e.PropertyName == nameof(PackageViewModel.HasEdit))
+            {
+                if (viewModel.HasEdit && _packageFileStream == null && _packageFileWatcher != null)
```

Comment:
Packages opened directly from package chooser are put into temp folder so they are not being watched and locked.